include/inviwo/core/interaction/trackballobject.h

```diff
@@ -40,27 +40,28 @@ namespace inviwo {
 class IVW_CORE_API TrackballObject {
 public:
     virtual ~TrackballObject() = default;
-    virtual const vec3& getLookTo() const = 0;
-    virtual const vec3& getLookFrom() const = 0;
-    virtual const vec3& getLookUp() const = 0;
+    [[nodiscard]] virtual vec3 getLookTo() const = 0;
```

Comment:
think we can disable the nodiscard warning. It seems excessive to add to every getter...